### PR TITLE
CLN: isort imports - io dir

### PR DESCRIPTION
**Pull request by alimcmaster1**
Signed-off-by: alimcmaster1 <alimcmaster1@gmail.com>

 Files in `pandas/io/*` ( 1 of 2)

- [x] xref #23334
- [ ] tests added / passed
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Review by WillAyd — APPROVED** [review]
lgtm if green

File: setup.cfg [hunk 1]
```diff
@@ -96,21 +96,6 @@ skip=
...
     pandas/util/_test_decorators.py,
-    pandas/io/s3.py,
-    pandas/io/parquet.py,
-    pandas/io/feather_format.py,
-    pandas/io/api.py,
-    pandas/io/sql.py,
-    pandas/io/clipboards.py,
-    pandas/io/excel.py,
-    pandas/io/date_converters.py,
-    pandas/io/testing.py,
```
Comment (WillAyd) [inline setup.cfg]: I guess as an extra callout this isn't in the diff, though I also don't see it existing as a module

Comment (alimcmaster1) [reply setup.cfg]: Good spot - my apologies I don't think this file ever existed in pandas running `git log --full-history  -- *pandas/io/testing.py` confirms this. I think I had a local file named this at one stage and it must have been present when I created the list of files to ignore.

**Comment by jreback** [comment]
thanks @alimcmaster1 

btw in the PR header, use xref issue number (and not closes) other GH will actually close the issue (IOW it doesn't understand partial)

**Pull request merged**